### DISCUSSION TITLE
middleware/kafka/user-guide/config/

### DISCUSSION THREAD

**Discussion by giscus** [DaoCloud/DaoCloud-docs#1714] [Announcements]
# middleware/kafka/user-guide/config/

DaoCloud Enterprise 5.0 下一代容器化平台集大成者，引领云原生浪潮，推动数字化转型。

https://docs.daocloud.io/middleware/kafka/user-guide/config/

**Comment by yayuntian** [comment]
立即生效，具体表示什么含义？会发生重启么？

**Reply by windsonsea** [reply]
是的，无需重启。PR #5308 已优化了表述。
